src/server/database/rethinkDriver.js

```diff
@@ -3,9 +3,11 @@ import getRethinkConfig from './getRethinkConfig';
 
 const config = getRethinkConfig();
 let driver;
-export default () => {
+const getRethink = () => {
```

Comment:
i'll rename this whole file in a separate PR. kinda bugs me that the export name doesn't match the file name.